CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+## 0.5.2:
+
+##### Breaking
+
+* None.
+
+##### Enhancements
+
+* Improve autocorrect for OpeningBraceRule.
+  [Yasuhiro Inami](https://github.com/inamiy)
+  [#274](https://github.com/realm/SwiftLint/issues/274)
```

Comment:
This should be the issue that's being resolved, not the PR number, as outlined in [CONTRIBUTING.md](https://github.com/realm/SwiftLint/blob/master/CONTRIBUTING.md).